ads/google/a4a/shared/test/test-shared-util.js

```diff
@@ -0,0 +1,59 @@
+import {validateAdContentRating} from "#ads/google/a4a/shared/shared-util";
+
+describes.sandboxed('validateAdContentRating', {}, function () {
+  it('should return true if valid content rating provided', function () {
+    const runTest = (contentRating) => {
+      expect(
+        validateAdContentRating(contentRating)
+      ).to.equal(true);
+    };
+    runTest(
+      /* contentRating= */ 'e',
+    );
+    runTest(
+      /* contentRating= */ 'pg',
+    );
+    runTest(
+      /* contentRating= */ 't',
+    );
+    runTest(
+      /* contentRating= */ 'ma',
+    );
+  });
+
+  it('should return false if invalid content rating provided', function () {
+    const runTest = (contentRating) => {
+      expect(validateAdContentRating(contentRating)).to.equal(false);
+    };
+    // One of pub control params is missing.
+    runTest(
+      "abcd"
+    );
+    runTest(
+      123
+    );
+    runTest(
+      "PG"
+    );
+  });
+
+// it('should reject invalid content ratings', function () {
```

Comment:
What is this commented out section?

Reply:
Test format if validation should throw an error. Assuming no error needed at this point?